core/src/components/datetime/datetime.tsx

```diff
@@ -318,6 +318,12 @@ export class Datetime implements ComponentInterface {
    */
   @Prop() showDefaultTimeLabel = true;
 
+  /**
+   * The time format of the `ion-datetime`. If no value is set, this is
+   * specified by the current locale.
+   */
+  @Prop() timeFormat?: '24h' | '12h' | null;
```

Comment:
Yes that correct, i named it '24h' because i think this sounds more logical for me as a user instead of `23` 🤔 Also the function is called `use24hour` so i thought it would be the correct naming. I understand the official hourCycle Reference but is this understandable for the users?

Reply:
I knew what you meant, but I also have only ever worked in `h12` and `h23` time. I think this may be confusing for developers who are looking `h24` support. Additionally, if we do add `h24` support this API becomes more confusing. Documentation is going to be crucial here, so I can help write that.

Reply:
Also given that these are the values used in browser APIs, I think these values will become more recognizable over time.

Reply:
Yes accepted 😃 Over time would be better if this matches the browser API